test/cyanide_binary_test.exs

```diff
@@ -0,0 +1,42 @@
+#
+# This file is part of Cyanide.
+#
+# Copyright 2019 Ispirata Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule CyanideBinaryTest do
+  use ExUnit.Case
+  import Cyanide.Binary
```

Comment:
`import` is used in very specific cases in Elixir since it makes you lose the information of where functions are coming from, by default you should instead use `alias` and then use only the last part of the module name (e.g. `Binary.subtype.int_to_atom(0)`.